packages/e2e/test/long-running/projector-ogmios-connection.test.ts

```diff
@@ -91,24 +92,14 @@ describe('projector ogmios connection', () => {
 
   afterEach(() => removePrevent());
 
-  it.skip('projector reconnects after a short delay', async () => {
+  it('projector reconnects after a short delay', async () => {
     await killContainer(ogmiosContainer);
     await waitProjector();
     // This throws because is not able to complete HTTP request
```

Comment:
This comment line **must** be removed as well since it says no longer the truth 😉

Reply:
[Removed](https://github.com/input-output-hk/cardano-js-sdk/pull/782/commits/c7faf01194561b2941c42c4a74517de0a5a9f7d9#diff-eda276c23f6ab654f368c7a2dd6a14410b791dda2843b155b35330c8970129a6R96) the line and squashed since it's just 1 change request